monkey/monkey_island/cc/repository/i_credentials_repository.py

```diff
@@ -0,0 +1,75 @@
+from abc import ABC
+from typing import Any, Sequence
+
+from monkey.cc.models import ConfiguredCredentials, StolenCredentials
```

Comment:
There is no class named `ConfiguredCredentials` and it probably wouldn't contain data that was any different than `StolenCredentials`. Can we just merge these into one class named `Credentials`?

Reply:
I was think about this. ATM `StolenCredentials` exists, maybe we can have `Credentials` with a type of `stolen` and `configured` ?

Reply:
I'm not sure why the object would need to track this metadata itself. The API provides a way to get stolen or configured credentials, so putting the metadata in the object is redundant.

Reply:
Previously all credentials were used from configuration. `StolenCredentials` were used in the report only, because they didn't contain secrets, just secret type (what kind of credentials were stolen).

Someone needs to keep track of which credentials were stolen and which were configured. I suppose we can trace-back stolen credentials from telemetries? But I think adding an `origin` attribute would also be fine for now.